index.js

```diff
@@ -50,6 +50,7 @@ const defaultOptions = {
   removeBlobs: true,
   fixInsertRule: true,
   skipThirdPartyRequests: false,
+  allowedThirdPartyDomains: [],
```

Comment:
`allowThirdPartyDomains` will be more consistent with other options, like `ignoreForPreload`.

Reply:
What about `includeThirdPartyRequests` instead of `allowedThirdPartyDomains `?
That's consistent with `option.include`.